docs/io-cloud-storage-sink.md

```diff
@@ -215,3 +232,15 @@ Before using the Cloud Storage sink connector, you need to create a configuratio
         The date is generated based on the `partitionerType` parameter in the configuration. And the `${messageSequenceId}` is generated by `FunctionCommon.getSequenceId(message.getMessageId())`.
     - Format suffix: `.parquet`
         This part is generated based on the `formatType` parameter in the configuration.
+    
+## Permissions
+
+### AWS S3 permission policies
+
+The suggested permission policies for AWS S3 are:
```

Comment:
I am not sure whether we can change this sentence to "It is recommended to configure the following AWS S3 permission policies:"